lib/stages/RawReader.hpp

```diff
@@ -252,22 +252,22 @@ RawReader<T>::RawReader(Config& config, const std::string& unique_name,
     if (local_dm && use_comet)
         FATAL_ERROR("Cannot use local dataset manager and dataset broker together.")
 
-    chunked = config.exists(unique_name, "chunk_size");
-    if (chunked) {
-        chunk_size = config.get<std::vector<int>>(unique_name, "chunk_size");
-        if (chunk_size.size() != 3)
-            throw std::invalid_argument("Chunk size needs exactly three "
-                                        "elements (has "
-                                        + std::to_string(chunk_size.size()) + ").");
-        chunk_t = chunk_size[2];
-        chunk_f = chunk_size[0];
-        if (chunk_size[0] < 1 || chunk_size[1] < 1 || chunk_size[2] < 1)
-            throw std::invalid_argument("RawReader: config: Chunk size "
-                                        "needs to be greater or equal to (1,1,1) (is ("
-                                        + std::to_string(chunk_size[0]) + ","
-                                        + std::to_string(chunk_size[1]) + ","
-                                        + std::to_string(chunk_size[2]) + ")).");
-    }
+    chunk_size = config.get_default<std::vector<int>>(unique_name, "chunk_size", {4, 64, 128});
+    chunked = true;
```

Comment:
How would you read use RawReader without chunking now?

Generally I think we shouldn't set default chunk sizes in either the transpose stages or here in the RawReader. It's not clear enough what a good default chunk size is for all cases.